metadata-ingestion/src/datahub/ingestion/source/metadata/business_glossary.py

```diff
@@ -30,6 +34,8 @@
 # This needed to map path presents in inherits, contains, values, and related_terms to terms' optional id
 path_vs_id: Dict[str, Optional[str]] = {}
 
+URN_RESERVED_CHARS = ("%",)
```

Comment:
please add to urn_encoder.py instead

You can add '%' to the `RESERVED_CHARS` var in that file (and also convert it from a list to a set)

Reply:
Done